src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/Networking/Libuv.cs

```diff
@@ -5,7 +5,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Networking
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking
 {
     public class LibuvFunctions
```

Comment:
Nit: I should have done this before, but rename the file to LibuvFunctions.cs. Make sure no other files are misnamed.